pylhc_submitter/job_submitter.py

```diff
@@ -350,10 +350,14 @@ def _create_jobs(
 
     if njobs == 0:
         raise ValueError(f"No (new) jobs found!")
-    if (njobs > HTCONDOR_JOBLIMIT) and not local_or_dry_run:
-        raise ValueError(f"Too many jobs! Allowed {HTCONDOR_JOBLIMIT}, given {njobs}.")
     if njobs > HTCONDOR_JOBLIMIT:
-        LOG.warning(f"You are running {njobs} jobs. Submission to HTCondor will not be possible as it exceeds the {HTCONDOR_JOBLIMIT} jobs limit. ")    
+        if local_or_dry_run:
+            LOG.warning(
+                f"You are running {njobs} jobs. Submission to HTCondor will not be possible as it exceeds "
+                f"the {HTCONDOR_JOBLIMIT} jobs limit."
+            )
+        else:
+            raise ValueError(f"Too many jobs! Allowed {HTCONDOR_JOBLIMIT}, given {njobs}.")

```

Comment:
I thought we were still discussing this? IMO we should not throw an error at all, but log the warning always (with "may not be possible") and remove the local_or_dryrun parameter again

Reply:
I'm not against it, but that removes the point of the "limit" entirely. Where do we get the 100k from? I do not see any mention in the batch docs.